sdx-submitter.go

```diff
@@ -0,0 +1,168 @@
+package main
+import(
+		"fmt"
+		"flag"
+		"github.com/streadway/amqp"
+		"os"
+		"io/ioutil"
+)
+
+func main() {
+
+	// Read a source file and place it on a rabbit topic exchange
+	var url, exchange, routing_key,    encryption_key,    signing_key, name, password string
+	var port, host , queue, vhost, message_path_and_file string
+	var msg_body []byte
+	var err error
+
+	getArguments(&name, &password, &port, &host, &url, &queue, &exchange, &routing_key, &encryption_key, &signing_key, &vhost, &message_path_and_file)
```

Comment:
Should really embed the reading of the flags into the `main` method - no need to factor out unless it's being used elsewhere.

Reply:
done